app/controllers/api/v1x0/actions_controller.rb

```diff
@@ -5,8 +5,6 @@ class ActionsController < ApplicationController
 
       def index
         req = Request.find(params.require(:request_id))
-        authorize req, :show?
```

Comment:
I feel leaving `authorize req`  here is better than moving it inside the `policy_scope`

Reply:
@bzwei When GraphQL makes a request the logic is similar to an index call, so its better to do the check in policy_scope in one place. From the pundit gems perspective all the index calls are scope based and that's where the check should happen for individual calls like show/update/edit/delete which work on a single object you can use authorize.

Reply:
GraphQL only does index from a model class, it does not pass has_many relation to the policy_scope. Look at the block https://github.com/RedHatInsights/approval-api/pull/313/files#diff-c423c5c790991b9cc7a73787c2cbe576R7, there we need to call `Request.find(params.require(:request_id))` again. It also checks permission and raise error which is already done by `authorize req`. We are re-inventing the logic but not utilizing the benefit from pundit.
Here the only line `authorize req, :show?` is simple and strait forward, easy to understand. GraphQL does not need this line anyway.